lib/widgets/position/position_right.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_internship/widgets/shapes/square.dart';
+
+class PositionOne extends StatelessWidget {
+  const PositionOne({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      alignment: Alignment.center,
+      children: [
+        Container(
+          width: 248,
+          height: 248,
+          decoration: BoxDecoration(
+            borderRadius: BorderRadius.circular(8),
+            color: Colors.blue,
+          ),
+        ),
+        const Positioned(
+          left: 192,
+          top: 96,
+          bottom: 96,
+          child: Square(
+            color: Colors.purple,
+            size: 56,
+          ),
```

Comment:
This is not the right way to do it. There is too much calculation and it is not generic.